<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>751</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>877</width>
    <height>753</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(188, 241, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>450</width>
     <height>700</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>700</height>
    </size>
   </property>
   <property name="pixmap">
    <pixmap>vm.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wallet">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(181, 181, 181);
color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(181, 181, 181);
color: rgb(255, 255, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Wallet</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>271</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(213, 229, 229);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="Controler" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 189, 255, 255), stop:1 rgba(89, 189, 255, 255));
border: 0px solid rgb(213, 229, 229);
background-color: (255, 79, 82);
border-style: outset;
border-width: 2px;
border-color: gray;
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="3">
          <widget class="QPushButton" name="Mng">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 12pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color:rgb(202, 135, 0);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>Manage</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 14pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(255, 79, 82);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="_0">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QPushButton" name="Enter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 14pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(0, 255, 127);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
	color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QPushButton" name="back">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(255, 79, 82);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 75 10pt &quot;MS Sellf Dlg 2&quot;;
	color: rgb(255,255,255);
	background-color: rgb(210, 211, 230);
	backgroung-color:rgb(191, 191, 191);
border-radious:35px;
}

QPushButton:hover{
color:rgb(61,61,61);
	backgroung-color:rgb(238,238,238);
}
QPushButton{
backgroung-color:rgb(208,0,0)
	
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(213, 229, 229);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="lable">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(213, 229, 229);
border: 0px solid rgb(213, 229, 229);</string>
                </property>
                <property name="text">
                 <string>Quantity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="display">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>25</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(213, 229, 229);
border: 0px solid rgb(213, 229, 229);</string>
                </property>
                <property name="text">
                 <string>Product code:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lable_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(213, 229, 229);
border: 0px solid rgb(213, 229, 229);</string>
                </property>
                <property name="text">
                 <string>Quantity</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="product">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 189, 255, 0), stop:1 rgba(89, 189, 255, 0));
border: 0px solid rgb(213, 229, 229);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>product/md.png</normaloff>product/md.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="Manage" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 189, 255, 0), stop:1 rgba(89, 189, 255, 0));</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>641</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>571</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>Product</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Product Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stok</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>442</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_syc">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>440</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
